jobserv/api/project.py

```diff
@@ -50,7 +50,7 @@ def project_update(proj):
     p = get_or_404(Project.query.filter_by(name=proj))
 
     d = request.get_json() or {}
-    if "allowed-hosts-tags" in d:
+    if "allowed-host-tags" in d:
```

Comment:
This looks like a typo fix of a previous commit. Squash?

Reply:
fixed in force push